src/stack.js

```diff
@@ -1,5 +1,55 @@
-'use strict'
-
 export default class Stack {
-  // your code here
+  constructor( data ) {
```

Comment:
Maybe you could have a default `data = { maxSize: Infinity }` so a stack can still be created using `new Stack()` with no arguments.